Buổi 11/homework.js

```diff
@@ -0,0 +1,115 @@
+//Bài 1
+const addElement =( a, number ) => {
+  const arr=[]
+  for(let i=0; i<number; i++){
+    arr.push(a)
+  }
+  return arr
+}
+
+//Bài 2
+const sortArr = ( a = [] ) => {
+  const newArr = []
+    for(let i=a.length-1; i >= 0; i--){
+        newArr.push(a[i])
+    }
+  return newArr
+}
+
+//Bài 3
+const check = ( arr = [] ) => {
+  const newArr = arr.filter( item => typeof(item) === 'number' && item !== 0)
+  console.log(newArr)   
+}
+
+
+//Bài 4
+const convertToObject = ( arr = [] ) => {
+  const x = Object.fromEntries(arr)
+  console.log(x)
+}
+
+//Bài 5
+const sortNumber = ( a = [] ) => {
+  a.sort()
+  return a 
+}
+
+
+//Bài 6
+const checkObject = data => {
+  const x = Object.prototype.toString.call(data) == "[object Object]"
+  return x

```

Comment:
Ví dụ lệnh này đúng
thì hãy viết là:
```suggestion
  return Object.prototype.toString.call(data) == "[object Object]"
```

và hãy thử: `checkObject(null)`